solutions/microfrontends/apps/docs/pages/about.tsx

```diff
@@ -13,11 +13,16 @@ export default function About() {
         <Code>apps/docs/pages/about.tsx</Code>).
       </Text>
       <Text>
-        Navigations between <Link href="/docs">Docs</Link> and{' '}
-        <Link href="/docs/about">About Docs</Link> are client-side transitions
+        Navigations between <Link href="/">Docs</Link> and{' '}
```

Comment:
Is it intentional that the `Link` and `a` tag have the same `href`?

Reply:
Yes! One of them is a same-link app driven by the base path and the other one takes you to the same app